aplib.net-demo/Assets/Scripts/Inventory/Key.cs

```diff
@@ -0,0 +1,18 @@
+public class Key : Item
+{
+    /// <summary>
+    /// The key ID, to check which door it can open
+    /// </summary>
+    public int id;
+    /// <summary>
+    /// the constructor that gives the key its Id
+    /// </summary>
+    /// <param name="constrId">the Id the key is given</param>
+    public Key(int constrId) => id = constrId;
+    private void Start()
```

Comment:
Note: I use the new naming of the member `Id` in this suggestion. For future notice; you don't have to call a parameter differently than the member variable. Taking the old code as an example: you can simply change `constrId` to `id` and assign the class member `id` by writing `this.id = id`. The `this` keyword denotes the class we are in.
```suggestion
    /// </summary>
    /// <param name="id">The ID the key is given.</param>
    public Key(int id) => Id = id;
    
    private void Start()
```

Reply:
Yea, ik weet wel hoe het werkt maar vind dat persoonlijk altijd een beetje verwarrend qua readability - maar heb het aangepast